### PR TITLE
Custom scan next

### PR DESCRIPTION
**Pull request by lenisko**

### CODE REVIEW

File: src/components/layout/FloatingBtn.jsx [hunk 1]
```diff
@@ -172,7 +172,7 @@ export default function FloatingButtons({
       ) : null}
       {perms?.scanner?.includes('scanZone') &&
       enableScanZone &&
-      scannerType === 'rdm' ? (
+      scannerType === 'rdm' || scannerType === 'custom' ? (
```
Comment (TurtIeSocks) [inline src/components/layout/FloatingBtn.jsx:175]:
```suggestion
      scannerType !== 'mad'  ? (
```

File: src/components/layout/dialogs/scanner/ScanNextTarget.jsx [hunk 2]
```diff
@@ -104,7 +104,7 @@ export default function ScanNextTarget({
                 {t('scan_next_choose')}
               </Typography>
             </Grid>
-            {scannerType === 'rdm' && (
+            {scannerType === 'rdm' || scannerType === 'custom' && (
```
Comment (TurtIeSocks) [inline src/components/layout/dialogs/scanner/ScanNextTarget.jsx:107]:
```suggestion
            {scannerType !== 'mad' && (
```

File: src/components/layout/dialogs/scanner/ScanZoneTarget.jsx [hunk 3]
```diff
@@ -152,7 +152,7 @@ export default function ScanZoneTarget({
                 {t('scan_zone_choose')}
               </Typography>
             </Grid>
-            {scannerType === 'rdm' && (
+            {scannerType === 'rdm' || scannerType === 'custom' && (
```
Comment (TurtIeSocks) [inline src/components/layout/dialogs/scanner/ScanZoneTarget.jsx:155]:
```suggestion
            {scannerType !== 'mad' && (
```

**Review by TurtIeSocks — CHANGES_REQUESTED** [review]
Overall looks good, thanks for the pr!

**Review by TurtIeSocks — APPROVED** [review]

**Comment by kamieniarz** [comment]
Looks like this PR broke discord messages

**Comment by lenisko** [comment]
@kamieniarz Unless you made some changes to `default.json` it shouldn't. Any errors?

**Comment by kamieniarz** [comment]
I changed nothing, nothing in logs too. Revert helped

**Comment by lenisko** [comment]
Tested by two all works. Make sure you haven't blocked changes to `default.json` or make sure you have this https://github.com/WatWowMap/ReactMap/blob/develop/server/src/configs/default.json#L565

**Comment by ReuschelCGN** [comment]
> Looks like this PR broke discord messages

Sorry but here too!

**Pull request merged**